### ISSUE TITLE
Get the patch version in the macOS OS inventory

### ISSUE THREAD

**Issue by chemamartinez** [wazuh/wazuh#6107]
## Description

Since #5860 the OS inventory will be used to detect vulnerabilities in macOS (see https://nvd.nist.gov/vuln/detail/CVE-2020-9859 as example).

As we can see in the link above, the macOS version includes the patch version (10.15.5). However, the patch is not gathered by Syscollector at https://github.com/wazuh/wazuh/blob/98eda96fb156aca87746178b9201d4e5a03683e8/src/shared/version_op.c#L637.

On the other hand, the OS information in macOS agents is read from a command output `sw_vers`. It should be replaced by parsing the following files:

- `/System/Library/CoreServices/SystemVersion.plist` in macOS
- `/System/Library/CoreServices/ServerVersion.plist` in macOS Server

Which follows the `plist` format:

```
~ cat /System/Library/CoreServices/SystemVersion.plist
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>ProductBuildVersion</key>
	<string>19B88</string>
	<key>ProductCopyright</key>
	<string>1983-2019 Apple Inc.</string>
	<key>ProductName</key>
	<string>Mac OS X</string>
	<key>ProductUserVisibleVersion</key>
	<string>10.15.1</string>
	<key>ProductVersion</key>
	<string>10.15.1</string>
	<key>iOSSupportVersion</key>
	<string>13.2</string>
</dict>
</plist>
```

## Tasks

- Modify `get_unix_version()` to read macOS OS info from the `plist` instead of running commands. This must be the default behavior, if the read fails, run the commands as always.
- Include the `patch` as a new field of the macOS OS inventory.
- Add this new field to the documentation https://documentation.wazuh.com/3.13/user-manual/capabilities/syscollector.html#operating-system.

**Comment by MarcelKemp** [comment]
Development branch: https://github.com/wazuh/wazuh/tree/6107-patch-version-macOS-inventory

**Comment by chemamartinez** [comment]
Merged! #6227
